client/src/client-only-routes/ShowCertification.js

```diff
@@ -102,9 +133,17 @@ const ShowCertification = props => {
       signedInUserName,
       isDonating,
       cert: { username = '' },
+      fetchProfileForUser,
+      user,
       executeGA
     } = props;
 
+    if (!signedInUserName || signedInUserName !== username) {
+      if (isEmpty(user)) {
```

Comment:
```suggestion
      if (isEmpty(user) && username) {
```
I noticed it would try and query like this, `/api/users/get-public-profile?username=`, which the api doesn't like.